com.unity.robotics.urdf-importer/Runtime/UrdfComponents/UrdfJoints/UrdfJointRevolute.cs

```diff
@@ -71,11 +55,7 @@ public override float GetVelocity()
         /// <returns>floating point in Nm</returns>
         public override float GetEffort()
         {
-#if UNITY_2020_1_OR_NEWER
             return unityJoint.jointForce[xAxis];
```

Comment:
Is there a corresponding API to read force ?

Reply:
I think this is related to the concept of inverse dynamics, which is only available starting 2022.1. @JuozasK

Reply:
Yeah, for now jointForce will only return what you set there before yourself. As for getting actual drive forces or related counteracting forces, that API landed in 2022.1.0a16

There's work on improving the Inverse Dynamics API with a accumulated forces and a external forces getter which would allow to get the force in reduced space needed to counteract any previously applied forces in general space

Reply:
I think if we were to solve this, someone should open a separate ticket. I'll leave this to Vidur. Clearly out of scope for the purposes of this PR.